src/components/SQForm/useForm.tsx

```diff
@@ -1,11 +1,45 @@
 import React from 'react';
-import {getIn, useField} from 'formik';
+import {
+  FieldHelperProps,
+  FieldInputProps,
+  FieldMetaProps,
+  getIn,
+  useField
+} from 'formik';
 import WarningIcon from '@material-ui/icons/NewReleases';
 import VerifiedIcon from '@material-ui/icons/VerifiedUser';
 
+interface useFormParam {
+  name: string;
+  isRequired: boolean;
+  onBlur?: React.FocusEventHandler;
+  onChange?: React.ChangeEventHandler;
+}
+
+interface useFormReturn {
+  formikField: {
+    field: FieldInputProps<any>;
```

Comment:
If we don't know what that type argument will be, `unknown` would be a better option - we lose basically all type safety with `any`.

I have a feeling that we could/should use a generic for the value rather than unknown or any, but I'd have to look more into that later